.github/workflows/build_and_test.yml

```diff
@@ -42,6 +42,7 @@ jobs:
 
     - name: Set environment variables
       run: |
+        echo _JAVA_OPTIONS= '-Djdk.util.zip.disableZip64ExtraFieldValidation=true' >> $GITHUB_ENV      
```

Comment:
can you please create an separate PR for this as this is happening due to  https://help.liferay.com/hc/en-us/articles/17904500197389-java-util-zip-ZipException-Invalid-CEN-header-invalid-zip64-extra-data-field-size-when-starting-dxp-using-JDK-11-0-20

more info
(https://www.oracle.com/java/technologies/javase/11-0-20-relnotes.html)
`java.util.zip.ZipFile has been updated to provide additional validation of ZIP64 extra fields when opening a ZIP file. This validation may be disabled by setting the system property jdk.util.zip.disableZip64ExtraFieldValidation to true.`